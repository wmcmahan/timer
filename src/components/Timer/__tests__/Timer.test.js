/**
 * @jest-environment jsdom 
 */
import React from 'react';
import { render } from '@testing-library/react';
import { Timer } from '../index';

describe('Timer', () => {
  it('renders standard Timer', () => {
    const { asFragment } = render(<Timer duration={5} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
