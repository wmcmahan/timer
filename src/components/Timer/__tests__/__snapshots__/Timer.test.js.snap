// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Timer renders standard Timer 1`] = `
<DocumentFragment>
  <div
    class="sc-bdnxRM haceqI"
  >
    <svg
      class="sc-gtsrHT jAithI"
    >
      <circle
        class="sc-dlnjwi MzNKe"
        r="90"
      />
      <circle
        class="sc-dlnjwi sc-hKFxyN MzNKe jUhpcP"
        r="90"
        stroke-dasharray="565.4866776461628"
        stroke-dashoffset="565.4866776461628"
      />
    </svg>
  </div>
</DocumentFragment>
`;
